### PR TITLE
[WIP] [#777] Add a build profile for Jaeger

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -346,6 +347,12 @@
         <version>${netty.tcnative.version}</version>
         <scope>runtime</scope>
       </dependency>
+      <!-- Jaeger support -->
+      <dependency>
+        <groupId>io.jaegertracing</groupId>
+        <artifactId>jaeger-client</artifactId>
```
Comment (sophokles73) [inline bom/pom.xml]: I think that we will need a works-with CQ for this, won't we?

Comment (ctron) [reply bom/pom.xml]: I guess this should be sufficient yes. I wouldn't really enable anything be default, as without a Jaeger server, it isn't really useful anyway.

**Comment by sophokles73** [comment]
@ctron I am not sure if we are on the same page here. My understanding is that we add a profile for adding the Jaeger Java client to the Docker images we build for our services. This PR is **not** about adding a Jaeger backend to the deployment scripts.

I am asking because the CQ you have created seems to indicate otherwise ...

**Comment by ctron** [comment]
Following up on the discussion from Eclipse Con … we indeed need a second works-with CQ for the Java Jaeger client. The reason for this CQ was the Jager agent, which I finally found in that repository: https://github.com/jaegertracing/jaeger/tree/master/cmd/agent … no too obvious from the root.

So I do think we need both. And I will create a CQ for the second one right now.

**Pull request merged**